pkg/providers/aws/cluster_network_provider_moq.go

```diff
@@ -10,14 +10,15 @@ import (
 )
 
 var (
-	lockNetworkManagerMockCreateNetwork            sync.RWMutex
-	lockNetworkManagerMockCreateNetworkConnection  sync.RWMutex
-	lockNetworkManagerMockCreateNetworkPeering     sync.RWMutex
-	lockNetworkManagerMockDeleteNetwork            sync.RWMutex
-	lockNetworkManagerMockDeleteNetworkConnection  sync.RWMutex
-	lockNetworkManagerMockDeleteNetworkPeering     sync.RWMutex
-	lockNetworkManagerMockGetClusterNetworkPeering sync.RWMutex
-	lockNetworkManagerMockIsEnabled                sync.RWMutex
+	lockNetworkManagerMockCreateNetwork               sync.RWMutex
```

Comment:
I did this moq change manually just to get things running. Can probably use the automation to ensure there's nothing missing here.

Reply:
Yep I'd probably run `go generate ./...` just to be sure